vitest.config.e2e.ts

```diff
@@ -0,0 +1,8 @@
+import { defineConfig } from "vitest/config";
+
+export default defineConfig({
+  test: {
+    include: ["lib/lib-dynamodb/**/*.e2e.spec.{ts,js}", "lib/lib-storage/**/*.e2e.spec.{ts,js}"],
+    environment: "node",
+  },
+});
```

Comment:
This is not part of CI. It is a temporary local-dev file for running vitest e2e tests from the application root.